### ISSUE TITLE
Internal Server Error on Configure Import page in staging environment

### ISSUE THREAD

**Issue by mtnorthrop** [verbb/formie#691]
**Description**
I get an Internal Server Error when I click on the "Configure Import" button in my staging environment.

The error in `web.log` reads:
```
2021-12-15 16:49:02 [-][1][d05bcs00gf1k98fjv4ch2r5tsm][error][Twig\Error\SyntaxError] Twig\Error\SyntaxError: Unknown "dump" function. in /MY_PROJECT_PATH/vendor/verbb/formie/src/templates/settings/import-export/import-configure.html:12
```

It looks like the problem is that the `dump()` function is being used in the `import-configure.html`, but this is part of the Twig DebugExtension, which probably isn't being loaded because my staging environment isn't a debug environment. This means the import function probably doesn't work in production environments either.

(I was trying to import a form that I had created in my dev environment into my staging environment so I wouldn't have to set it up again manually.)

**Additional info**
- Plugin version: 1.5.2
- Craft version: 1.7.25.1

**Comment by engram-design** [comment]
Ah, good point - I was trying to display the potential errors of an import, but that tag obviously doesn't exist when `devMode` is off -  whoops.

Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by mtnorthrop** [comment]
Thanks for the fix! No worries, I've done the same a million times myself. :-)

**Comment by engram-design** [comment]
Fixed in [1.5.3](https://github.com/verbb/formie/releases/tag/1.5.3)
